docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Why We Created the Site Showcase
+date: 2018-06-15
+author: Shannon Soper
+tags: [""]
+---
+ 
+We recently finished the first phase of the [Site Showcase](https://next.gatsbyjs.org/showcase/), which is a searchable, filterable library of sites built with Gatsby!
+
+![Featured Sites](featured-sites.png)
+
+![Filtered Sites](filtered-sites.png)
+
+## Submitting your site
+
+If you’d like to submit a site to the showcase, please follow the [instructions in this document](https://github.com/gatsbyjs/gatsby/blob/v2/docs/docs/site-showcase-submissions.md).
+ 
+# Why build a site showcase?
+
+We built the showcase to make it easier for you to find the source code for Gatsby sites and easier to show off what Gatsby can do to potential clients / coworkers.
+
+## Source code
+
+Many people learn how to build things by looking at src code for other sites. As you browse through the site showcase, look for the GitHub icon on each site’s detail page which links you to the source code (not all sites will have it):
+
+![Site Showcase Source Code Available Icon](site-showcase-source-code.png)
+
+## Showing off what Gatsby can do
+
+The site showcase can help you pitch Gatsby to clients and coworkers as the best option for their next project. As they search the site showcase, it will provide a vision of what developers can accomplish with Gatsby.
+
+# UX research process
+
+Here’s where we discussed and built the site showcase if you'd like to go through the discussion that led to this project:
+* [Site Showcase Issue](https://github.com/gatsbyjs/gatsby/issues/4392)
+* [Site Showcase WIP PR](https://github.com/gatsbyjs/gatsby/pull/5524)
+
+## Empathy map
+
+Here’s a summary of what I learned from over 80 interviews with Gatsby users. This "empathy map" describes what kinds of pain points we're trying to solve with the site showcase.
+
+![Empathy Map Site Showcase](empathy-map-site-showcase.jpg)
+ 
+## User Flow
+
+The empathy map reflects how it took most people a lot of time browsing through the Awesome Gatsby list to find example sites and how many people share their sites through Twitter and there isn’t enough time or room for everyone to get retweeted by the @Gatsbyjs account.
+
+With the empathy map and the interviews as our guide, we created a list of jobs to be done and imagined the typical flow a user would take through the site showcase. We did this to make sure we planned both a sensible information architecture and the necessary functionality to meet user’s main needs.
+
+![User flow](user-flow-site-showcase.jpg)
+ 
+## Defining the MVP
+
+Here are the features that clearly needed to be included in the MVP; you'll notice that we checked all the boxes thanks to the hard work of many people!
```

Comment:
Is the list too long to call everyone out by name? If we can specifically give credit to folks, that's always best.

Reply:
ok cool! I wanted to, just haven't got this feedback before and wondered whether people would be too shy. Just by GitHub name, I assume?